app/views/info/_lead_metrics.html.erb

```diff
@@ -1,16 +1,33 @@
 <section id="lead-metrics">
   <% if lead_metrics %>
+    <%- if format == 'guide' %>
+    <h2>Metrics for entire guide</h2>
```

Comment:
This could be made more generic with `<h2>Metrics for entire <%= format%></h2>` or `Metrics across all parts`?